docs/en/operations/settings/merge-tree-settings.md

```diff
@@ -623,6 +623,19 @@ Possible values:
 
 Default value: false
 
+## number_of_free_entries_in_pool_to_execute_optimize_entire_partition {#number_of_free_entries_in_pool_to_execute_optimize_entire_partition}
+
+When there is less than specified number of free entries in pool, do not execute optimizing entire partition in the background (this task generated when set `min_age_to_force_merge_seconds` and enable `min_age_to_force_merge_on_partition_only`). This is to leave free threads for regular merges and avoid "Too many parts".
+
+Possible values:
+
+- Positive integer.
+
+Default value: 20
```

Comment:
It means that background "optimize" has the same priority as mutations. But it should have lower priority, so the default value should be higher